packages/webpack-cli/lib/webpack-cli.js

```diff
@@ -780,17 +843,13 @@ class WebpackCLI {
                 let loadedCommand;
 
                 try {
-                    loadedCommand = require(pkg);
+                    loadedCommand = await this.tryRequireThenImport(pkg, false);
```

Comment:
why false here and default true in function?

Reply:
Because we ignore errors in some case, we have tests on this